applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -972,54 +941,49 @@ void UPwSmallStrainElement<TDim, TNumNodes>::CalculateMassMatrix(MatrixType& rMa
 
     const IndexType N_DOF = this->GetNumberOfDOF();
 
-    if (rMassMatrix.size1() != N_DOF)
-        rMassMatrix.resize(N_DOF, N_DOF, false);
+    if (rMassMatrix.size1() != N_DOF) rMassMatrix.resize(N_DOF, N_DOF, false);
     noalias(rMassMatrix) = ZeroMatrix(N_DOF, N_DOF);
 
     const GeometryType& rGeom = this->GetGeometry();
-    const GeometryType::IntegrationPointsArrayType& IntegrationPoints = rGeom.IntegrationPoints(this->mThisIntegrationMethod);
+    const GeometryType::IntegrationPointsArrayType& IntegrationPoints =
+        rGeom.IntegrationPoints(this->mThisIntegrationMethod);
     const IndexType NumGPoints = IntegrationPoints.size();
 
     ElementVariables Variables;
-    this->InitializeElementVariables(Variables,
-                                     rCurrentProcessInfo);
+    this->InitializeElementVariables(Variables, rCurrentProcessInfo);
 
     // Create general parameters of retention law
     RetentionLaw::Parameters RetentionParameters(this->GetProperties(), rCurrentProcessInfo);
 
     // Defining shape functions at all integration points
     // Defining necessary variables
-    BoundedMatrix<double, TDim+1, TNumNodes*(TDim+1)> Nut              = ZeroMatrix(TDim+1, TNumNodes*(TDim+1));
-    BoundedMatrix<double, TDim+1, TNumNodes*(TDim+1)> AuxDensityMatrix = ZeroMatrix(TDim+1, TNumNodes*(TDim+1));
-    BoundedMatrix<double, TDim+1, TDim+1>             DensityMatrix    = ZeroMatrix(TDim+1, TDim+1);
+    BoundedMatrix<double, TDim + 1, TNumNodes*(TDim + 1)> Nut = ZeroMatrix(TDim + 1, TNumNodes * (TDim + 1));
```

Comment:
As far as I can found, it's either spaces around any calculation (so also all 'normal' formulas), or remove the spaces everywhere, which I think makes it less readable. My theory what happens with the * here, is that clang-format does not know what the type is of TNumNodes. Therefore, it might think TNumNodes* is a pointer to something and therefore it does not add the extra space. At least for '-' and '/' it also adds the spaces. '*' seems to be the only exception